gdpr/anonymizers/__init__.py

```diff
@@ -0,0 +1,19 @@
+from .fields import (
+    AccountNumberFieldAnonymizer, CharFieldAnonymizer, DateFieldAnonymizer, DecimalFieldAnonymizer, FunctionAnonymizer,
+    IPAddressFieldAnonymizer, StaticValueAnonymizer)
+from .generic_relation import ReverseGenericRelationAnonymizer
+from .gis import GISPointFieldAnonymizer
+from .hash_fields import HashTextFieldAnonymizer, MD5TextFieldAnonymizer, SHA256TextFieldAnonymizer
+from .legacy_fields import (
+    DummyFileAnonymizer, EmailFieldAnonymizer, IDCardDataFieldAnonymizer, NameFieldAnonymizer,
+    PersonalIIDFieldAnonymizer, PhoneFieldAnonymizer, UsernameFieldAnonymizer)
+from .model_anonymizers import DeleteModelAnonymizer, ModelAnonymizer
+
+__all__ = [
```

Comment:
osobně bych používal tuple místo listu, ale je to jedno. Nicméně bych nahradil "  za  '